src/tests/ftest/osa/offline_drain.yaml

```diff
@@ -7,7 +7,7 @@ hosts:
     - server-C
   test_clients:
     - client-D
-timeout: 1100
+timeout: 1200
```

Comment:
The test is timing out in Ci may need more time?

Reply:
In general, the overall test time for different object class testing has increased. So, just adding some more time for handling EC class testing. Present, timeout issue with respect to offline drain is related to DAOS-8609. Di is looking at the issue and feels there is a product issue.

Reply:
So does the test need to be skipped or wait until the defect is resolved?

Reply:
Let me disable the test for now.